### PR TITLE
Add Rollback and Inactive env

### PR DESCRIPTION
**Pull request by danielburnley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,10 +16,34 @@ We strongly advise creating a new IAM user just for Route53 just in case.
 
 ### Retrieving currently live domain
 ```
-/bin/current_live_environment
+env PYTHONPATH=. /bin/current_live_environment
```
Comment (davidwinter) [inline README.md]: Is this `fish` shell specific? Might be worth mentioning that in the README.

Comment (danielburnley) [reply README.md]:
```
sh-4.4$ env PYTHONPATH=. bin/rollback
Flipping live to test-blue.example.co.uk.
```

Comment (danielburnley) [reply README.md]: @davidwinter ^

File: bin/rollback [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        print(FlipDNSRecord(
+            client=aws_client,
+            live=current_live,
+            blue=os.getenv('BLUE_DEPLOY_DOMAIN'),
+            green=os.getenv('GREEN_DEPLOY_DOMAIN'),
+            target_domain = os.getenv('TARGET_DEPLOY_DOMAIN')
+        ).call()['ChangeInfo']['Comment'])
+
+    except RuntimeError as error:
+        print(format(error))
```
Comment (davidwinter) [inline bin/rollback]: Is there much difference in here compared to the flip command? Is there any way to reduce the duplication or just reuse the existing script?

Comment (danielburnley) [reply bin/rollback]: We're keeping the scripts self contained for now

**Pull request merged**